lib/os.js

```diff
@@ -121,6 +121,43 @@ function endianness() {
 }
 endianness[Symbol.toPrimitive] = () => kEndianness;
 
+function networkInterfaces() {
+  const interfaceAddresses = getInterfaceAddresses();
+
+  return Object.entries(interfaceAddresses).reduce((acc, [key, val]) => {
+    acc[key] = val.map((v) => {
+      const protocol = v.family.toLowerCase();
+
+      return Object.assign({}, v, {
```

Comment:
style nit: assign `getCIDRSuffix(v.netmask, protocol)` to a const, and do this in one line.